test/Select-test.js

```diff
@@ -2389,6 +2389,69 @@ describe('Select', function() {
 			});
 		});
 
+		describe('searchingText', function () {
+
```

Comment:
Note this function padding will end up failing happiness once the [bug](https://github.com/feross/standard/issues/170) is fixed in eslint.